### DISCUSSION TITLE
专有名词的英文缩写和简中译名的平衡

### DISCUSSION THREAD

**Discussion by bixudamajiang** [liuli-moe/to-the-stars#116] [General]
有很多原文就是写缩写的名词或翻译出来过于长的名词会得到缩写的译名，但我个人认为需要统一一下译文语言的选择。比如说可信计算框架（TCF）和魔法少女行会（MSY）。在一些地方用的是英文缩写，在一些地方用的却是简中译名，还有些地方是中英结合。建议统一一下格式以降低多语的阅读难度，个人建议在一些地方保留双语乃至更多类型的文字，降低阅读门槛。

**Comment by rxliuli** [comment]
> 有很多原文就是写缩写的名词或翻译出来过于长的名词会得到缩写的译名，但我个人认为需要统一一下译文语言的选择。比如说可信计算框架（TCF）和魔法少女行会（MSY）。在一些地方用的是英文缩写，在一些地方用的却是简中译名，还有些地方是中英结合。建议统一一下格式以降低多语的阅读难度，个人建议在一些地方保留双语乃至更多类型的文字，降低阅读门槛。

抱歉没有看到。一般来说吾辈习惯专有名词在第一次出现的标识英文原文，之后就可以直接使用译文。

**Reply by bixudamajiang** [reply]
> > 有很多原文就是写缩写的名词或翻译出来过于长的名词会得到缩写的译名，但我个人认为需要统一一下译文语言的选择。比如说可信计算框架（TCF）和魔法少女行会（MSY）。在一些地方用的是英文缩写，在一些地方用的却是简中译名，还有些地方是中英结合。建议统一一下格式以降低多语的阅读难度，个人建议在一些地方保留双语乃至更多类型的文字，降低阅读门槛。
> 
> 抱歉没有看到。一般来说吾辈习惯专有名词在第一次出现的标识英文原文，之后就可以直接使用译文。

这个问题也不是很大，只是统一上比较困难，等完结了再说吧
